### ISSUE TITLE
Outdated files in the dev-1.21-ko.4 branch

### ISSUE THREAD

**Issue by seokho-son** [kubernetes/website#28288]
# This is a Bug Report
## Problem
Outdated files in the upstream/dev-1.21-ko.4 branch.
This report is based on `scripts/diff_l10n_branches.py ko upstream/dev-1.21-ko.3 upstream/dev-1.21-ko.4`

### 10 files to be modified 
1. [ ] M1.  `content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md`  | 1(+XS) 1(-)
1. [ ] M2.  `content/en/docs/concepts/services-networking/network-policies.md`  | 2(+XS) 2(-)
1. [ ] M3.  `content/en/docs/concepts/workloads/pods/disruptions.md`  | 1(+XS) 1(-)
1. [ ] M4.  `content/en/docs/reference/command-line-tools-reference/kube-proxy.md`  | 1(+XS) 1(-)
1. [ ] M5.  `content/en/docs/reference/scheduling/config.md`  | 1(+XS) 1(-)
1. [ ] M6.  `content/en/docs/setup/best-practices/cluster-large.md`  | 2(+XS) 2(-)
1. [ ] M7.  `content/en/docs/setup/production-environment/_index.md`  | 3(+XS) 3(-)
1. [ ] M8.  `content/en/docs/setup/production-environment/container-runtimes.md`  | 16(+S) 6(-)
1. [ ] M9.  `content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md`  | 68(+M) 22(-)
1. [ ] M10.  `content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md`  | 1(+XS) 1(-)

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `upstream/dev-1.21-ko.4` branch.

For example:
```
# checkout `upstream/dev-1.21-ko.4`
...
# check what is updated in the upstream 
git diff upstream/dev-1.21-ko.3 upstream/dev-1.21-ko.4 -- content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md
# apply changes to content/ko
vi content/ko/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md
...
# commit and push
...
# make PR to `upstream/dev-1.21-ko.4`
```

## Pages to Update

Pages in content/ko

**Comment by seokho-son** [comment]
- M1 ~ M5 : @seokho-son

**Comment by pjhwa** [comment]
/assign @pjhwa for M6 ~ M10

- M1 ~ M5 : @seokho-son 
- M6 ~ M10 : @pjhwa

**Comment by jihoon-seo** [comment]
This issue is resolved by #28470 .
/close
